workflow-examples/src/main/java/com/redhat/parodos/examples/prebuilt/migrationtoolkit/MTAAnalysisReport.java

```diff
@@ -0,0 +1,70 @@
+package com.redhat.parodos.examples.prebuilt.migrationtoolkit;
+
+import java.util.Iterator;
+import java.util.Scanner;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class MTAAnalysisReport {
+
+	/**
+	 * migrationIssuesReport is assumed to be valid html. MTA 6.1 should supply a CSV as
```

Comment:
the comment refers to `migrationIssuesReport` but it is nowhere to be found in this PR.
could you update the comment and perhaps use javadoc style?